src/genie/libs/parser/linux/route.py

```diff
@@ -88,3 +91,56 @@ def cli(self, flag=None, output=None):
                 continue
 
         return parsed_dict
+
+
+# =======================================================
+# Parser for 'netstat -rn'
+# =======================================================
+class NetstatRn(RouteSchema):
```

Comment:
I have now added parser for netstat -rn command that extends the route schema.
I am not happy with the class name class NetstatRn(). any sugestiens?

Reply:
how about "ShowNetworkStatusRoute"